config.js

```diff
@@ -42,7 +42,7 @@ module.exports = {
      * you do not plan on using the SDK.  It is, however, strongly recommended
      * you use the SDK.
      */
-    twofactorUrl: 'https://mfa.bandwidth.com/app/two-factor',
+    twofactorUrl: (accId, endpoint) => {return 'https://mfa.bandwidth.com/api/v1/accounts/' + accId + '/code/' + endpoint},
```

Comment:
I would make this anonymous function three lines, with normal break on the curly braces, and a semicolon at the end of the line.

Reply:
I actually prefer this being single line.  As the content of this file is mainly "configuration" and this function being a single liner return statement.

Reply:
ok, just the missing semicolon then